### PR TITLE
Structure Json

### LINKED ISSUE #37: Structure JSON
The JSON should have a schema, something that's structured.

### PR DESCRIPTION
**Pull request by Ian-Stewart-Binks**
Fixes #37

### CODE REVIEW

**Comment by Ian-Stewart-Binks** [comment]
The scraper still doesn't work, however.

File: util/scrape.py [hunk 1]
```diff
@@ -48,15 +48,18 @@ def _get_tables(self):
         table_one.extend(table_two)
         return table_one
 
-    def _build_rule_dict(self, html):
+    def _build_rules(self, html):
         """Build the tree of rules as a dictionary"""
```
Comment (Ian-Stewart-Binks) [inline util/scrape.py:52]: Need to change docs.

File: util/scrape.py [hunk 2]
```diff
@@ -48,15 +48,18 @@ def _get_tables(self):
...
-        rule_dict = {}
-        for i in html:
-            i = self._cleanup_string(i)
+        rules = []
+        clean_html = map(self._cleanup_string, html)
```
Comment (emmettu) [inline util/scrape.py:54]: Why do you iterate over html once to clean it, and then again to build the rules?

Comment (Ian-Stewart-Binks) [inline util/scrape.py:54]: I find this easier to read.

**Comment by Anirudh94** [comment]
So following our chat on gitter about manually adding rules, there's no more point to this script. I'm going to close this PR. 
On that note we should look into removing the python script + flake8 checks.

**Pull request closed without merging**